postreise/plot/plot_utilization_map.py

```diff
@@ -35,92 +20,109 @@ def map_risk_bind(
+    :param tuple(int, int) figsize: size of the bokeh figure (in pixels).
+    :param bool show_color_bar: whether to render the color bar on the figure.
+    :param dict plot_states_kwargs: keyword arguments to be passed to
+        :func:`postreise.plot.plot_states.plot_states`.
+    :return: (*bokeh.plotting.figure*) -- map of lines with risk and bind incidents
+        color coded.
     """
-    if us_states_dat is None:
-        us_states_dat = get_state_borders()
-
     if risk_or_bind == "risk":
         risk_or_bind_units = "Risk (MWH)"
 
     if risk_or_bind == "bind":
         risk_or_bind_units = "Binding incidents"
 
+    if palette is None:
+        palette = list(traffic_palette)
+
     # projection steps for mapping
     branch_congestion = pd.concat(
         [branch.loc[congestion_stats.index], congestion_stats], axis=1
     )
     branch_map_all = project_branch(branch)
-    multi_line_source_all = ColumnDataSource(
-        {
-            "xs": branch_map_all[["from_x", "to_x"]].values.tolist(),
-            "ys": branch_map_all[["from_y", "to_y"]].values.tolist(),
-            "cap": branch_map_all["rateA"] / 2000 + 0.2,
-        }
-    )
-    a, b = project_borders(us_states_dat)
-    tools: str = "pan,wheel_zoom,reset,save"
 
     branch_congestion = branch_congestion[branch_congestion[risk_or_bind] > 0]
     branch_congestion.sort_values(by=[risk_or_bind])
     branch_map = project_branch(branch_congestion)
     min_val = branch_congestion[risk_or_bind].min() if vmin is None else vmin
     max_val = branch_congestion[risk_or_bind].max() if vmax is None else vmax
     mapper = linear_cmap(
-        field_name=risk_or_bind, palette=traffic_palette, low=min_val, high=max_val
-    )
-    color_bar = ColorBar(
-        color_mapper=mapper["transform"],
-        width=385 if is_website else 500,
-        height=5,
-        location=(0, 0),
-        title=risk_or_bind_units,
-        orientation="horizontal",
-        padding=5,
+        field_name=risk_or_bind, palette=palette, low=min_val, high=max_val
     )
     multi_line_source = ColumnDataSource(
         {
             "xs": branch_map[["from_x", "to_x"]].values.tolist(),
             "ys": branch_map[["from_y", "to_y"]].values.tolist(),
             risk_or_bind: branch_map[risk_or_bind],
             "value": branch_map[risk_or_bind].round(),
-            "cap": branch_map["capacity"] / 1000 + 2,
+            "cap": (
+                branch_map["capacity"] * select_branch_scale_factor
+                + select_branch_min_width
+            ),
             "capacity": branch_map.rateA.round(),
         }
     )
 
     # Set up figure
     p = figure(
-        tools=tools,
+        tools="pan,wheel_zoom,reset,save",
         x_axis_location=None,
         y_axis_location=None,
-        plot_width=800,
-        plot_height=800,
+        plot_width=figsize[0],
+        plot_height=figsize[1],
         output_backend="webgl",
-        sizing_mode="stretch_both",
```

Comment:
I would recommend to `sizing_mode` to "scale_both" to ensure that the plot is scaled to original aspect ratio base on width. I had to use to make sure that the plot has the same width than the notebook cells or the webpage width (without having to play with `figsize`)

Reply:
Done.